orchestra/utils/models.py

```diff
@@ -8,6 +9,27 @@ class DeleteMixin(object):
     Intended to be used with models that have an `is_deleted` field.
     """
     def delete(self, *args, **kwargs):
+        # Implement cascading deletes.
+        # NOTE(lydia): This ignores any other constraints passed to on_delete.
+        fields = self._meta.get_fields(include_hidden=True)
```

Comment:
https://docs.djangoproject.com/en/1.9/ref/models/meta/#migrating-from-the-old-api